cns/NetworkContainerContract.go

```diff
@@ -218,6 +218,8 @@ type HostIPInfo struct {
 
 type IPConfigRequest struct {
 	DesiredIPAddress    string
+	PodInterfaceID      string
```

Comment:
Can you also populate these two values otherwise we will have to take another CNI rollout